tracing/README.md

```diff
@@ -39,6 +39,15 @@ change in configuration.
 The following tracing back-ends are known to work with Go kit through the
 OpenTracing interface and are highlighted in the [addsvc] example.
 
+## OpenCensus
```

Comment:
looks like the wrong place in the readme, doesn't work with the previous sentence.

Reply:
you are right, moving the section